<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0" columnstretch="0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QCustomPlot" name="plotISS" native="true">
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>350</width>
         <height>691</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QCustomPlot" name="plot" native="true">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>280</y>
          <width>291</width>
          <height>241</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Dialog">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>640</y>
          <width>135</width>
          <height>40</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Sprawdź czas przelotu 
 nad danym punktem</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_VelocityISS">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>60</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>50</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Prędkość:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>111</x>
          <y>201</y>
          <width>63</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Współrzędne</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_LatitudeISS">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>222</y>
          <width>47</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_LongitudeISS">
        <property name="geometry">
         <rect>
          <x>165</x>
          <y>222</y>
          <width>47</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>43</x>
          <y>120</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Wysokość:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_AltitudeISS">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>120</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>250</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>[km]</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>220</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>[t]</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>411</width>
         <height>691</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/pobierz.png);</string>
       </property>
       <widget class="QPushButton" name="pushButton_Panel">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>0</y>
          <width>60</width>
          <height>60</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/images/menu.jpg);
border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_Center">
       <property name="geometry">
        <rect>
         <x>1100</x>
         <y>520</y>
         <width>34</width>
         <height>22</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Path">
       <property name="geometry">
        <rect>
         <x>1110</x>
         <y>560</y>
         <width>34</width>
         <height>22</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_View">
       <property name="geometry">
        <rect>
         <x>1110</x>
         <y>600</y>
         <width>34</width>
         <height>22</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="chFlight" native="true">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>160</y>
         <width>411</width>
         <height>321</height>
        </rect>
       </property>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>104</width>
          <height>70</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_2">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>110</y>
          <width>104</width>
          <height>70</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="check_button">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>280</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Sprawdź</string>
        </property>
       </widget>
       <widget class="QLabel" name="des_label">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Podaj współrzędne przelotu</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="long_label">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>80</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Szerokość</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lat_label">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>80</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Długość</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="close_button">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>20</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="chFlightOP" native="true">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>40</y>
         <width>121</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
      <zorder>pushButton_Center</zorder>
      <zorder>pushButton_Path</zorder>
      <zorder>pushButton_View</zorder>
      <zorder>widget_2</zorder>
      <zorder>widget</zorder>
      <zorder>chFlightOP</zorder>
      <zorder>chFlight</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
